### DISCUSSION TITLE
I need to integrate junit-report results back to jira based on the ticket number in jira

### DISCUSSION THREAD

**Discussion by seidoo99** [cypress-io/cypress#24744] [General]
hello everyone. I'm trying to generate a custom XML report for my cypress tests so they can map directly to a specific issue type in Xray for Jira upon import. I've read through a lot of the documentation on reporters and I've set up the configuration to generate XML files for tests/test runs so currently I can generate a combined report of my whole test suite, but when I import the file into Xray it creates new issuekeys. I already have created the test cases in Xray and I want my XML file to contain specific information on which cypress test should go to each Xray test case. Does anyone have advice or know how to do this?

**Comment by delboy1978uk** [comment]
if you are using the cucumber preprocessor for feature files then you can annotate each test with the ticket number. for plain cypress tests, you can use the cypress xray plugin, and you just put the ticket number in the `it('ABC-XXXX does something')`
